src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.exceptions.DukeException;
+import duke.exceptions.IncorrectNumberException;
+import duke.tasks.TaskList;
+
+public class DoneCommand extends Command {
+    private final int taskNum;
```

Comment:
I think constant names are supposed to be all uppercase with an underscore to separate words? Unless I misread and this actually isn't a constant oops